src/core/components/auth/authorization-popup.jsx

```diff
@@ -38,7 +38,9 @@ export default class AuthorizationPopup extends React.Component {
                                   errSelectors={ errSelectors }
                                   authSelectors={ authSelectors }
                                   authActions={ authActions }
-                                  specSelectors={ specSelectors }/>
+                                  specSelectors={ specSelectors }
```

Comment:
I'd prefer to give AuthorizationPopup an `onAuthorize` callback here that does the refetching work, rather than passing in _more_ system dependencies.

Reply:
ok, no problem